client/src/actions/portfolioActions.js

```diff
@@ -17,7 +17,7 @@ export const addTransaction = (expData, history) => dispatch => {
     .catch(err =>
       dispatch({
         type: GET_ERRORS,
-        payload: err.response.data
```

Comment:
why should you not send out the error, if it did come up??

Reply:
good point, will resolve in next commit